ramls/request.json

```
@@ -282,6 +282,10 @@
     "itemLocationCode": {
       "description": "Allow specifying item location when creating title-level requests",
       "type": "string"
+    },
+    "isDcbReRequestCancellation": {
+      "description": "Indicates if the request was cancelled during a DCB transaction update. This determines whether the Kafka event will be handled and if an email notification will be sent",
```

I don't think that mentioning the interpretation of this flag is a good idea
```suggestion
      "description": "Indicates if the request was cancelled during a DCB transaction update",
```